editor/theme/templates/west-referral/referrer/share.tsx

```
@@ -0,0 +1,81 @@
+      setBusy(false);
+    });
+  };
+
+  return (
+    <Drawer {...props}>
+      <DrawerContent>
+        <div className="mx-auto w-full">
+          <article
+            className="p-4 prose prose-sm dark:prose-invert"
+            dangerouslySetInnerHTML={{
+              __html: data?.article?.html ?? "",
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)